api/ai/new/social_algorithm/social_graph.py

```diff
@@ -56,5 +57,7 @@ def create_social_graph(students: List[Student]):
             social_graph[(student.id, other.id)] = Relationship.DEFAULT.value
             if other.id in student.relationships:
                 # update to real relationship value if one exists
-                social_graph[(student.id, other.id)] = student.relationships[other.id]
+                social_graph[(student.id, other.id)] = get_relationship_value(
```

Comment:
I don't understand the need for introducing `get_relationship_value` in the actual algorithm code? 

I realize the error I made here (I think), but wouldn't it always be `social_graph[(student.id, other.id)] = student.relationships[other.id].value`
Can we not rely on Student.relationships to be Dict[str, Relationship]?